tests/compiler/test_plugins.py

```diff
@@ -239,3 +242,16 @@ def test_explicit_plugin_exception(snippetcompiler):
         assert e.__cause__.message == "Test: " + msg
     except Exception as e:
         assert False, "Expected ExplicitPluginException, got %s" % e
+
+
+def test_plugin_load_exception(snippetcompiler):
+    module: str = "test_plugin_load_error"
+    snippetcompiler.setup_for_snippet(f"import {module}")
+    modpath = os.path.join(os.path.dirname(os.path.dirname(__file__)), "data", "modules", module)
+    expected: str = (
+        "Unable to load all plug-ins for module test_plugin_load_error:"
+        "\n\tNameError while loading plugin module inmanta_plugins.test_plugin_load_error.invalid_code:"
```

Comment:
I am wondering whether we should remove the `inmanta_plugins` part from the path. The fact that we put all the modules in the package `inmanta_plugins` is an implementation details and is irrelevant for end-users, isn't it. @wouterdb What do you think?

Reply:
I get your point but I disagree with your conclusion. It's more than an implementation detail since it's exposed as an actual package that can be used by modules in plugin files. It's also mentioned in the docs (I assume) so it's something a model developer could (should?) be aware of.

Reply:
I see it's not in the docs, only in the `pytest-inmanta` documentation. So I guess it depends on whether we mean this only for testing or also for use in plugins (in which case we should create a ticket to document it). I'm in favor of the second. I think it's actually used like that already in dialog.

Reply:
Indeed. You use it if you want to import something defined in another module. In that case it's relevant. You can ignore this comment.